src/WebJobs.Extensions.DurableTask/Bindings/ObjectValueProvider.cs

```diff
@@ -38,7 +38,7 @@ public Task<object> GetValueAsync()
 
         public string ToInvokeString()
         {
-            return this.value.ToString();
+            return (this.value != null) ? this.value.ToString() : string.Empty;
```

Comment:
I don't remember what `ToInvokeString()` is used for, though I believe @MikeStall told me at one point. I thought maybe it was just for manual execution in the portal, or for the old monitoring system. Do you know?

Also, had you considered simply returning null instead of an empty string? The problem with converting null into an empty string generally is that there are some cases where `null` and `string.Empty` are not meant to be equivalent, and could in theory cause unexpected behavior.

Reply:
Also, do other bindings in Azure Functions implement the same kind of null handling behavior?

Reply:
@cgillum, I am not sure. @MikeStall can probably provide more insights to your questions above.
 
I have updated the PR to return null.